scripts/extensions/sams/src/utils/assets.ts

```diff
@@ -141,3 +149,18 @@ export function isAssetLockedByCurrentUser(asset: Partial<IAssetItem>): boolean
         return false;
     }
 }
+
+export function toClientFormat(response: IAutoTaggingSearchResult): OrderedMap<string, IAssetTag> {
```

Comment:
Now that this function is used in multiple places, a more descriptive name would help. As we're converting a search result to Tags, maybe something like `convertTagSearchResultToAssetTags`?